packages/wells/src/__tests__/integration/wells.int.spec.ts

```diff
@@ -41,6 +42,27 @@ describeIfCondition('CogniteClient setup in wells - integration test', () => {
     /* eslint-enable */
   });
 
+  test('get source assets for well', async () => {
+    expect(client).not.toBeUndefined();
+    const wellId: number = 5432591169464385;
+    const well: Well | undefined = await client.wells.getById(wellId)
+    const sources: Asset[] = await well?.sourceAssets() as Asset[]
+    expect(sources).not.toBeUndefined();
+    expect(sources?.length).toBe(1);
+    const source = sources[0];
```

Comment:
nit: Instead of casting, could null handling be a better approach?: 
```suggestion
    const sources: Asset[] | undefined = await well?.sourceAssets()
    expect(sources).not.toBeUndefined();
    expect(sources?.length).toBe(1);
    const source = sources![0];
```